glslc/README.asciidoc

```diff
@@ -110,13 +115,41 @@ Otherwise,
 
 `--help` tells the glslc compiler to display all available options and exit.
 
+==== `--show-limits`
+
+`--show-limits` shows default resource limits for shader compilation.  The syntax
+is the same as accepted by `-flimit=` and for the contents of the file specified
+by `-flimit-file`.   This option accepts Glslang resource configuration files,
+e.g. as emitted by `glslangValidator -c`.
+
 ==== `-o`
 
 `-o` lets you specify the output file's name. It cannot be used when there are
 multiple files generated. A filename of `-` represents standard output.
 
 === Language and Mode Selection Options
 
+[[option-f-entry-point]]
+==== `-fentry-point=`
+
+`-fentry-point=<name>` lets you specify the entry point name.  This is only
+significant for HLSL compilation.  The default is "main".
+
+[[option-f-limit]]
+==== `-flimit=`
+
+`-flimit=<resource-limits>` lets you specify resource limits for the shader.
+The argument should be a sequence of limit name, integer value pairs.  Tokens
+should be separated by whitespace.  Use `--show-limits` to show the default values,
+and example syntax.
```

Comment:
Does this affect shaders specified before it? Or does it behave similarly to `-fshader-stage`?